cmd/bootstrap/cmd/block.go

```diff
@@ -34,7 +32,7 @@ func constructRootBlock(rootHeader *flow.Header, setup *flow.EpochSetup, commit
 		Receipts:   nil,
 		Results:    nil,
 		// TODO: shortcut in bootstrapping; we will probably have to start with a non-empty KV store in the future
```

Comment:
```suggestion
```
We do still need to do this, but it no longer will necessitate changes to this part of the code.